flutter_barcode_reader_web/lib/barcode_web.dart

```diff
@@ -0,0 +1,235 @@
+@JS()
+library jsqrscanner;
+
+import 'dart:async';
+import 'dart:html';
+
+import 'package:flutter/services.dart';
+import 'package:js/js.dart';
+import 'package:js/js_util.dart';
+import 'package:flutter_web_plugins/flutter_web_plugins.dart';
+
+class BarcodeScanPlugin {
+  Completer<String> _completer;
+
+  static void registerWith(Registrar registrar) {
+    final MethodChannel channel = MethodChannel(
+      'de.mintware.barcode_scan',
+      const StandardMethodCodec(),
+      registrar.messenger
+    );
+    final BarcodeScanPlugin instance = new BarcodeScanPlugin();
+    channel.setMethodCallHandler(instance.handleMethodCall);
+  }
+
+  Future<String> handleMethodCall(MethodCall call) async {
+    _createCSS();
+    var script = document.createElement('script');
+    script.setAttribute('type', 'text/javascript');
+    document.querySelector('head').append(script);
+    script.setAttribute('src', 'https://cdn.jsdelivr.net/gh/jbialobr/JsQRScanner@master/war/js/jsqrscanner.nocache.js');
```

Comment:
External JavaScript should be embedded in the plugin.

Reply:
I have moved the script to a local js file embedded in the plugin, as well as extracted the CSS styles into its own separate file. Oddly enough, previous attempts of embedding the JS file failed, but now it's all working fine